README.md

```diff
@@ -56,10 +56,10 @@ The support for HF models is provided by our HF model adapter. One can obtain si
 
 ```python
 # fms model
-llama: LLaMA = LLaMA(config)
+llama = get_model("llama", "13b")
```

Comment:
I'm a bit confused by the intent of the (original) example since this doesn't load weights

Reply:
Was just simply showing it was creating a LLaMA model, not necessarily that it was loading the weights. Just an uninitialized LLaMA model